src/main/kotlin/org/jetbrains/research/testspark/helpers/psi/java/JavaPsiHelper.kt

```diff
@@ -173,18 +165,18 @@ class JavaPsiHelper(private val psiFile: PsiFile) : PsiHelper {
             log.info(
                 "The test can be generated for: \n " +
                     " 1) Class ${javaPsiClassWrapped.qualifiedName} \n" +
-                    " 2) Method ${javaPsiMethodWrapped.methodDescriptor}" +
+                    " 2) Method ${javaPsiMethodWrapped.name} \n" +
                     " 3) Line $line",
             )
         }
 
         return result.toArray()
     }
 
-    override fun getLineDisplayName(line: Int): String = "<html><b><font color='orange'>line</font> $line</b></html>"
+    override fun getLineDisplayName(line: Int) = "<html><b><font color='orange'>line</font> $line</b></html>"
 
     override fun getClassDisplayName(psiClass: PsiClassWrapper): String =
-        (psiClass as JavaPsiClassWrapper).getClassDisplayName()
+        "<html><b><font color='orange'>${psiClass.classType.representation}</font> ${psiClass.qualifiedName}</b></html>"
```

Comment:
Do we always want to return it using HTML syntax? When I see this API, I expect only the name that I can use in display and not the whole thing with HTML.